js/ui/toolbar.js

```diff
@@ -294,7 +294,6 @@ const Toolbar = ToolbarBase.inherit({
                 break;
             case 'visible':
```

Comment:
You can remove this `case`. It duplicates the default case now

Reply:
You are right. I missed it. Fixed in https://github.com/DevExpress/DevExtreme/pull/17910/commits/231815965c48049375d3af177441bc9462af5fe1